docs/contributing/authoring-polyfills.html

```diff
@@ -155,7 +155,7 @@ <h2 id="browsers">Browsers</h2>
 			</thead>
 			<tbody>
 			<tr><td><code>ie</code></td><td>Internet Explorer / Edge / Edge Mobile</td><td>{{baselines.ie}}</td></tr>
-			<tr><td><code>ie_mob</code></td><td>Internet Explorer Mobile</td><td>{{baselines.ie_mob}}</td></tr>
+			<tr><td><code>ie_mob</code></td><td>Internet Explorer / Edge Mobile</td><td>{{baselines.ie_mob}}</td></tr>
```

Comment:
Yeah, actually you're right, the mobile/desktop split is ambiguous.  Let me take another look at that.